platform/src/util/errors.ts

```diff
@@ -0,0 +1,15 @@
+export interface ApolloError {
+  errorCode: number;
+  errorType: string;
+  errorMessage: string;
+  errorDetails?: Record<string, any>;
+}
+
+export function isApolloError(value: any): value is ApolloError {
```

Comment:
Can we just say `if (value.statusCode)` ? Then any object coming out of python with an errorCode will be treated as an error

I think that's simple and robust enough for our needs

Reply:
You can keep this util function though

Reply:
Just simplify it